corehq/apps/accounting/models.py

```diff
@@ -679,12 +679,9 @@ def new_rate(cls, feature_name, feature_type,
         feature, _ = Feature.objects.get_or_create(name=feature_name, feature_type=feature_type)
         rate = FeatureRate(feature=feature)
 
-        if monthly_fee is not None:
-            rate.monthly_fee = monthly_fee
-        if monthly_limit is not None:
-            rate.monthly_limit = monthly_limit
-        if per_excess_fee is not None:
-            rate.per_excess_fee = per_excess_fee
+        rate.monthly_fee = monthly_fee or rate.monthly_fee
```

Comment:
All of these are datetime objects. So assuming these should either be datetime values or None. and won't be [] or "" or False or other python's Falsy.